sdk/go/pulumi/context.go

```diff
@@ -70,7 +71,7 @@ func NewContext(ctx context.Context, info RunInfo) (*Context, error) {
 	if addr := info.EngineAddr; addr != "" {
 		conn, err := grpc.Dial(info.EngineAddr, grpc.WithInsecure())
 		if err != nil {
-			return nil, errors.Wrap(err, "connecting to engine over RPC")
+			return nil, fmt.Errorf("connecting to engine over RPC: %s", err)
```

Comment:
Should these be `%w` to maintain the wrapping semantics?  Not sure how important that is - but potentially keeps things closer to where they were before?

Reply:
Yes - assuming we're happy with supporting only Go 1.13 and above (which I am), using `%w` here is better.